examples/typescript/src/Character.tsx

```diff
@@ -1,26 +1,30 @@
 import * as React from 'react';
 import {
-  GetCharacter,
-  GetCharacter_hero_friends,
-} from './__generated__/GetCharacter';
+  GetCharacterQuery,
+  GetCharacterQueryVariables,
+  Episode,
+} from './__generated__/types';
+import { GetCharacter as QUERY } from './queries';
 import { Query, QueryResult } from 'react-apollo';
-// import Query, { QueryResult } from '../../../src/Query';
 
-const QUERY = require('./Character.graphql');
-// https://github.com/Microsoft/TypeScript/issues/6395#issuecomment-282133254
-class CharacterQuery extends Query<GetCharacter> {}
+class CharacterQuery extends Query<
+  GetCharacterQuery,
+  GetCharacterQueryVariables
+> {}
 
 export interface CharacterProps {
-  episode: string;
+  episode: Episode;
 }
 
 export const Character = (props: CharacterProps) => {
```

Comment:
`Character` should be explicitly typed as `React.StatelessComponent`.

Reply:
good catch